src/com/tictacgo/data/Piece.java

```diff
@@ -149,6 +140,14 @@ public int getXPosition() {
 		return position[0];
 	}
 
+  public int getRow() {
+    return position[0] + 1;
```

Comment:
This is going to get us into trouble someday...

It actually already did for me once. We should consider using {0, 1, 2} everywhere.